src/cli/dcos_docker/commands/create.py

```diff
@@ -78,6 +78,56 @@ def _validate_docker_network(
         raise click.BadParameter(message=message)
 
 
+def _validate_port_map(
+    ctx: click.core.Context,
+    param: Union[click.core.Option, click.core.Parameter],
+    value: Any,
+) -> Dict[str, Any]:
```

Comment:
I think that `Any` should be `int`, here and in other places in this PR.

Reply:
Fixed